src/controlSurface.cpp

```diff
@@ -354,6 +359,28 @@ class Surface: public IReaperControlSurface {
 		}
 	}
 
+	void reportTimeSelectionWhilePlaying(double playPos) {
+		double start, end;
+		GetSet_LoopTimeRange(false, false, &start, &end, false);
+		double startDiff = playPos - start;
+		double endDiff = playPos - end;
+		if (start == end)
+			return;
```

Comment:
nit: Avoid unnecessary work if we return early.

```suggestion
		if (start == end) {
			return;
		}
		double startDiff = playPos - start;
		double endDiff = playPos - end;
```